test/libc/fmt/fmt_test.c

```diff
@@ -64,6 +64,10 @@ TEST(fmt, d) {
   EXPECT_STREQ("-2147483648", _gc(xasprintf("% d", INT_MIN)));
 }
 
+TEST(fmt, s) {
+  EXPECT_STREQ("Wide character output test", _gc(xasprintf("%S", L"Wide character output test")));
```

Comment:
This line looks long, please see the style guide https://github.com/jart/cosmopolitan/blob/master/CONTRIBUTING.md#style-guide In your own test files you can `/* clang-format off */` and use your own styling.

Reply:
Ah, sorry, forgot to run `clang-format` on it. Thanks for fixing that!